BasicUNet features: (6, 16, 32, 64, 128, 16).
Training ...
  0%|                                                     | 0/1 [00:00<?, ?it/s]
X: torch.Size([2, 159, 291, 514]) y: torch.Size([1, 159, 291, 514])
  0%|                                                     | 0/1 [00:00<?, ?it/s]Exception in thread ChkStopThr:
Traceback (most recent call last):
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/threading.py", line 973, in _bootstrap_inner
Exception in thread NetStatThr:
Traceback (most recent call last):
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/threading.py", line 910, in run
    self.run()
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 166, in check_status
    self._target(*self._args, **self._kwargs)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 148, in check_network_status
    status_response = self._interface.communicate_stop_status()
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 114, in communicate_stop_status
    status_response = self._interface.communicate_network_status()
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 125, in communicate_network_status
    resp = self._communicate_stop_status(status)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 378, in _communicate_stop_status
resp = self._communicate_network_status(status)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 388, in _communicate_network_status
    resp = self._communicate(req, local=True)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 213, in _communicate
    resp = self._communicate(req, local=True)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 213, in _communicate
        return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 218, in _communicate_async
return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 218, in _communicate_async
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
    raise Exception("The wandb backend process has shutdown")
